src/Options.js

```diff
@@ -31,10 +32,20 @@ const sizeMultipliers = [
     0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.2, 1.44, 1.728, 2.074, 2.488,
 ];
 
-const sizeAtStyle = function(size, style) {
+const sizeAtStyle = function(size: number, style: StyleInterface): number {
     return style.size < 2 ? size : sizeStyleMap[size - 1][style.size - 1];
 };
 
+export type OptionsData = $Shape<{
```

Comment:
I don't think we should use `$Shape`.  This will require all props on `OptionData` to be marked as optional, e.g. `color?: string` which is okay imo b/c in a couple of places we're missing some existence checks.